scripts/hostapd/install

```diff
@@ -4,84 +4,32 @@ set -e
 
+if ! [ "$(which confget)" ]; then
+   sudo apt-get install confget
+fi
+
 # Install hostapd and dnsmasq to run IEEE 802.11 Access Point
-if ! [ "$(which hostapd)" ] || ! [ "$(which dnsmasq)" ]; then
-    sudo apt-get install hostapd dnsmasq radvd -y
+if ! [ "$(which hostapd)" ] || ! [ "$(which dnsmasq)" ] || ! [ "$(which iptables)" ]; then
+    sudo apt-get install hostapd dnsmasq radvd iptables -y
 fi
 
-# Configure wlan0 interface
-sudo cp /etc/network/interfaces /etc/network/interfaces.backup
-START=$(grep -n "allow-hotplug wlan0" /etc/network/interfaces | awk '{ print $1 }' FS=':')
-END=$(expr $START + 3)
-sudo sed -i "${START},${END}d" /etc/network/interfaces
-echo "" | sudo tee --append /etc/network/interfaces > /dev/null
-echo "allow-hotplug wlan0" | sudo tee --append /etc/network/interfaces > /dev/null
-echo "iface wlan0 inet static" | sudo tee --append /etc/network/interfaces > /dev/null
-echo "    address 10.0.0.1" | sudo tee --append /etc/network/interfaces > /dev/null
-echo "    netmask 255.255.255.0" | sudo tee --append /etc/network/interfaces > /dev/null
-echo "    network 10.0.0.0" | sudo tee --append /etc/network/interfaces > /dev/null
-echo "    broadcast 10.0.0.255" | sudo tee --append /etc/network/interfaces > /dev/null
-echo "iface wlan0 inet6 static" | sudo tee --append /etc/network/interfaces > /dev/null
-echo "    address fdfc::2" | sudo tee --append /etc/network/interfaces > /dev/null
-echo "    netmask 64" | sudo tee --append /etc/network/interfaces > /dev/null
+APSSID=$(sudo grep -m 1 '"ipv6"' /etc/cjdroute.conf | awk '{ print $2 }' | sed 's/[",]//g' | sed "s/.*:/$MESH_NAME-/g")
 
-# Enable packet forwarding
-sudo cp /etc/sysctl.conf /etc/sysctl.conf.backup
-sudo sed -i 's/#net.ipv4.ip_forward=1/net.ipv4.ip_forward=1/' /etc/sysctl.conf
-sudo sed -i 's/#net.ipv6.conf.all.forwarding=1/net.ipv6.conf.all.forwarding=1/' /etc/sysctl.conf
+sudo cp $BASE_DIR/95-wlan-hostap.link /etc/systemd/network/95-wlan-hostap.link
+sudo sed -i "s/##SSID##/$APSSID/g" "/etc/systemd/network/95-wlan-hostap.link"
 
-# Get network name and password
-APSSID=$(sudo grep -m 1 '"ipv6"' /etc/cjdroute.conf | awk '{ print $2 }' | sed 's/[",]//g' | sed "s/.*:/$MESH_NAME-/g")
+sudo cp $BASE_DIR/95-wlan-hostap.network /etc/systemd/network/95-wlan-hostap.network
 
-# Select wpa-eap or wpa-psk
-read -p "Use WPA-EAP (Y) or WPA2-PSK (n) for WiFi Access Point $APSSID? " -n 1 -r
-echo ""
-if [[ $REPLY =~ ^[Nn]$ ]]; then
-    USE_EAP=false
-    while [ "${#APPASS}" -lt 8 ] || [ "${#APPASS}" -gt 63 ]; do
-        read -p "Set WPA2-PSK password (8-63 characters): " APPASS;
-    done
-else
-    USE_EAP=true
-    while [ "${#APPASS}" -lt 8 ] || [ "${#APPASS}" -gt 63 ]; do
-        read -p "Set WPA-EAP password (8-63 characters) for user \"guest\": " APPASS;
-    done
-fi
+sudo cp $BASE_DIR/50-hostapd.rules /etc/udev/rules.d/50-hostapd.rules 
 
-# Configure network with hostapd
-sudo cp "$BASE_DIR/nat.sh" /etc/hostapd/nat.sh
-if [ "$USE_EAP" = true ]; then
```

Comment:
Why is all EAP stuff deleted?

Reply:
Just for now to get the basics working then we can cherry pick the rest of it in.

Reply:
AP tests where successfull and now EAP has now been readded